config/webhook/certs/tls.crt

```diff
@@ -0,0 +1,24 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
This needs to be either generated or referenced (e.g. referencing the autogenerated certs on OpenShift and reference certmanager's certificate on plain Kubernetes).